promgen/views.py

```diff
@@ -1033,8 +1033,15 @@ def get_queryset(self):
             return self.queryset.filter(
                 Q(alertlabel__name="Service", alertlabel__value__contains=search)
                 | Q(alertlabel__name="Project", alertlabel__value__contains=search)
+                | Q(alertlabel__name="Job", alertlabel__value__contains=search)
             )
-        return self.queryset
+
+        qs = self.queryset
+        for key, value in self.request.GET.items():
+            if key == "page" or key == "search":
```

Comment:
Could probably write this as `if key in ["page", "search"]`

Reply:
fixed